addons/website_forum/tests/test_performance.py

```diff
@@ -19,7 +20,7 @@ def setUpClass(cls):
             'name': 'TestQuestion',
             'content': "Who's afraid of Virginia Wolf?",
             'forum_id': cls.forum.id,
-            'tag_ids': [(0, 0, {'name': 'Tag2', 'forum_id': cls.forum.id})]
+            'tag_ids': [Command.create({'name': 'Tag2', 'forum_id': cls.forum.id})]
```

Comment:
Should be done in first commit (even if Command is deprecated, you may use numbers)